src/rtc_stream.cpp

```diff
@@ -321,6 +337,32 @@ std::size_t rtc_stream_impl::read_data(char const *data, std::size_t size)
     return bytes_read;
 }
 
+size_t rtc_stream_impl::write_data(std::size_t size)
```

Comment:
@arvidn The maximum message size is now detected, and this internal method should do to write a chunk of given size, with sendBuffer accepting a collection of buffers to prevent a copy.